<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AvtorizationToAdminIn</class>
 <widget class="QDialog" name="AvtorizationToAdminIn">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>650</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Вход в админ панель</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin" stdset="0">
    <number>9</number>
   </property>
   <item>
    <widget class="QFrame" name="tbnAddTerminalDataAvtorization">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <property name="spacing">
       <number>3</number>
      </property>
      <property name="margin" stdset="0">
       <number>3</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="leftMargin">
         <number>3</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="lblInformation1Reg">
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Вход в админ панель</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="80">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="5,35,13" columnminimumwidth="0,0,0">
          <property name="leftMargin">
           <number>100</number>
          </property>
          <property name="rightMargin">
           <number>100</number>
          </property>
          <property name="spacing">
           <number>20</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="lblLoginToAdmin">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Логин:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="lblPasswordToAdmin">
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string>Пароль:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="editLoginReg">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="editPasswordReg">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="echoMode">
             <enum>QLineEdit::Password</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="2" rowspan="2">
           <widget class="QPushButton" name="btnEnterAdmin">
            <property name="focusPolicy">
             <enum>Qt::NoFocus</enum>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../../ekiosk.qrc">
              <normaloff>:/assets/icons/reg_9.png</normaloff>:/assets/icons/reg_9.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>120</width>
              <height>120</height>
             </size>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="wgtKeyPud" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <property name="margin" stdset="0">
       <number>1</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="layoutWgtKeyPud"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editLoginReg</tabstop>
  <tabstop>editPasswordReg</tabstop>
 </tabstops>
 <resources>
  <include location="../../ekiosk.qrc"/>
 </resources>
 <connections/>
</ui>
